test/1_stdlib/NewString.swift

```diff
@@ -114,6 +114,10 @@ func nonASCII() {
   var nsSliceUTF16 = newNSUTF16[i2..<i8] as NSString
   print("  \(repr(nsSliceUTF16))")
 
+  // CHECK-NEXT: fast C-String contents: nil
+  let cntgStr = unsafeBitCast(nsSliceUTF16, to: _NSContiguousString.self)
```

Comment:
…oh, maybe not, because the static hierarchy is different. `_unsafeReferenceCast`?